apps/extension/src/routes/popup/approval/origin.tsx

```diff
@@ -100,13 +100,16 @@ export const OriginApproval = () => {
             </div>
           </div>
         </div>
-        <ApproveDeny
-          variants={['gradient']}
-          approve={approve}
-          deny={deny}
-          ignore={lastRequest && ignore}
-          wait={3}
-        />
+
+        <div className='flex grow flex-col justify-end'>
```

Comment:
Pushes the Approve/Deny buttons all the way to the bottom of the window without having to use `position: fixed`